flow/e2e/bigquery/peer_flow_bq_test.go

```diff
@@ -767,6 +768,14 @@ func (s PeerFlowE2ETestSuiteBQ) Test_Types_BQ() {
 	// check if JSON on bigquery side is a good JSON
 	err = s.checkJSONValue(dstTableName, "c17", "sai", "-8.021390374331551")
 	require.NoError(s.t, err)
+
+	// check if HSTORE on bigquery side is a good JSON
+	err = s.checkJSONValue(dstTableName, "c46", "key1", "\"value1\"")
+	require.NoError(s.t, err)
+	err = s.checkJSONValue(dstTableName, "c46", "key2", "\"\"")
```

Comment:
`checkJSONValue` needs to be updated to accept nils, maybe should take `interface{}` a value instead of `string`